resources/packs/azure/azure.lr

```diff
@@ -608,7 +608,7 @@ private azure.subscription.storageService.account.tableService.properties {
 }
 
 // Azure Storage Account Service Properties Metrics 
-private azure.subscription.storageService.account.service.properties.metrics {
+private azure.subscription.storageService.account.service.properties.metrics @defaults("id includeAPIs enabled") {
```

Comment:
I am not sure how to test this one 🤔

Reply:
https://github.com/mondoohq/cnquery/blob/main/resources/packs/azure/azure.lr#L529

Reply:
I think this requires extra setup of logging for the bucket

Reply:
For sanity, I think we should just use the `id` for now until we can fully test. 

```suggestion
private azure.subscription.storageService.account.service.properties.metrics @defaults("id") {
```